### PR TITLE
C357057 Check that "Search/Filter" panel on the  "View all" page is saved collapsed state after returning from another application. (folijet) (TaaS)

### PR DESCRIPTION
**Pull request by AliaksandrShuleika**
[testrail](https://foliotest.testrail.io/index.php?/cases/view/357057)
[report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5486/allure/#suites/228a2a482430b8d58362c69c13c08344/33c9f728ca33dc08/)

### CODE REVIEW

File: cypress/support/fragments/data_import/logs/logsViewAll.js [hunk 1]
```diff
@@ -481,4 +483,10 @@ export default {
   noLogResultsFound: () => {
     cy.expect(logsResultPane.find(HTML('No results found. Please check your filters.')).exists());
   },
+
+  collapseButtonClick: () => cy.do(collapseButton.click()),
+
+  expandButtonClick: () => cy.do(expandButton.click()),
+
+  checkSearchPaneCollapsed: () => cy.expect(Pane('Search & filter').absent()),
```
Comment (dnovikau) [inline cypress/support/fragments/data_import/logs/logsViewAll.js:491]:
```JS 
Pane('Search & filter')
```

is already used in another method, please move it to constant and replace all usages

**Review by dnovikau — APPROVED** [review]

File: cypress/e2e/data-import/log-details/search-panel-collapsed.cy.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      });
+    });
+
+    it(
+      'C357057 Check that "Search/Filter" panel on the  "View all" page is saved collapsed state after returning from another application. (folijet) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+      () => {
+        // #1 Go to the "Data import" app
+        // The "Data import" page is displayed
+        cy.login(user.username, user.password, {
```
Comment (dnovikau) [inline cypress/e2e/data-import/log-details/search-panel-collapsed.cy.js:36]: I'd recommend to move login action to precondition

**Review by dnovikau — APPROVED** [review]

File: cypress/support/fragments/data_import/logs/logsViewAll.js (hunk 1 above)
Comment (AliaksandrShuleika) [reply cypress/support/fragments/data_import/logs/logsViewAll.js]: fixed

File: cypress/e2e/data-import/log-details/search-panel-collapsed.cy.js (hunk 2 above)
Comment (AliaksandrShuleika) [reply cypress/e2e/data-import/log-details/search-panel-collapsed.cy.js]: fixed

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**